src/plugins/koops_event.conf

```diff
@@ -30,23 +30,23 @@ EVENT=post-create type=Kerneloops remote!=1
 #EVENT=report_Kerneloops type=Kerneloops
         reporter-kerneloops
 
-EVENT=report_Bugzilla type=Kerneloops
+EVENT=report_Bugzilla type=Kerneloops mce!=non-fatal mce!=fatal
         reporter-bugzilla -b \
                 -F /etc/libreport/plugins/bugzilla_format_kernel.conf
 
 # Send micro report
-EVENT=report_uReport type=Kerneloops
+EVENT=report_uReport type=Kerneloops mce!=non-fatal mce!=fatal
```

Comment:
This behavior is expected but I was referring to system logs that appear in journald after a new MCE is detected. In such a case, ABRT runs `abrt-action-ureport` from `abrt-action-notify` (EVENT=notify) and this can cause unnecessary error log messages. Thinking about it a bit more, `abrt-action-notify` may ignore `EVENT=report_uReport` declaration and execute `abrt-action-ureport` or `reporter-ureport` directly.

Could you please double-check that there are no error logs in journald after a new MCE is detected and `abrt-action-notify` does not ignore your configuration changes (ie. ARBT does not send the MCE problem to FAF)?